src/NetCDF.jl

```diff
@@ -249,7 +255,7 @@ for (t,ending,arname) in funext
     @eval nc_get_var1_x(ncid::Integer,varid::Integer,start::Vector{UInt},::Type{$t})=begin $fname1(ncid,varid,start,$(arsym)); $(arsym)[1] end
 end
 
-function nc_get_vara_x!{T<:Union{UTF8String,ASCIIString}}(ncid::Integer,varid::Integer,start::Vector{UInt},count::Vector{UInt},retvalsa::Array{T})
+function nc_get_vara_x!(ncid::Integer,varid::Integer,start::Vector{UInt},count::Vector{UInt},retvalsa::Array{AbstractString})
```

Comment:
Can you loosen this again by making the function parametric? This way an Array{UTF8String} would not be accepted by the signature, so we should use

    function nc_get_vara_x!{T<:AbstractString}(....retvalsa::Array{T})